tetris/index.js

```diff
@@ -0,0 +1,121 @@
+var express = require('express');
+var app = express();
+var http = require('http').Server(app);
+var io = require('socket.io')(http);
+var path = require('path');
+var util = require('util');
```

Comment:
Consider `const util = require('util');` , if you are supporting only greenfield browsers, you can use good es6 features.
Node examples only use var for compatibility reasons.